README.md

```diff
@@ -152,3 +152,9 @@ Normal | 1 GPU | 2 GPUs | 3 GPUs | 4 GPUs
 -------|-------|--------|--------|-------
   870s |  912s |  477s  |  329s  |  254s
  1.00x | 0.95x | 1.82x  | 2.65x  | 3.42x
+
+
+## Attention
+Install ordering:
+1. [NCCL](https://github.com/NVIDIA/nccl) should be installed first.
+2. [libgpuarray/ pygpu](https://github.com/Theano/libgpuarray) should be installed after the installation of NCCL.
```

Comment:
Good idea of adding this information, but why at the end of this file? Can you move the note about nccl to the installation section of this file? I think this way, more people will see it when they need it.